src/main/java/com/griddynamics/cd/nrp/internal/model/config/NexusServer.java

```diff
@@ -28,16 +28,25 @@
 @RequiredArgsConstructor
 @XmlAccessorType(XmlAccessType.FIELD)
 public class NexusServer {
-    @Getter
     @NonNull
     @XmlElement(name = "url")
     private String url;
-    @Getter
     @NonNull
     @XmlElement(name = "user")
     private String user;
-    @Getter
```

Comment:
You could use this plugin https://plugins.jetbrains.com/plugin/6317 to resolve "missing" code.